### PR TITLE
[Docs] Remove Drift chat bot

### PR DESCRIPTION
**Pull request by samiahmedsiddiqui**

### CODE REVIEW

File: docs/netlify.toml [hunk 1]
```diff
@@ -1382,8 +1382,6 @@
             https://www.google.com/recaptcha/
             https://www.googletagmanager.com/
             https://www.gstatic.com/recaptcha/
-            https://x.clearbitjs.com
```
Comment (premkumr) [inline docs/netlify.toml:1385]: @samiahmedsiddiqui - why has the clearbitjs been removed ?

Comment (samiahmedsiddiqui) [reply docs/netlify.toml:1385]: @premkumr We are no longer using Clearbit directly. We integrated it from the different way by which we don't need this URL in our security policy headers.

I found few more instances of clearbit in security headers, if you like I can create a separate PR for that.

Comment (samiahmedsiddiqui) [reply docs/netlify.toml:1385]: @premkumr clearbitjs added back in this PR. I am creating a separate PR to remove clearbit headers.

**Review by premkumr — APPROVED** [review]
LGTM

**Pull request merged**